docs/app/documentation/component-docs/date-picker/examples/date-picker-i18n-example.component.ts

```diff
@@ -52,7 +52,7 @@ export class CustomI18nLabels extends CalendarI18nLabels {
 
 @Component({
     selector: 'fd-datepicker-i18n-example',
-    template: `<fd-date-picker [(ngModel)]="selectedDay"></fd-date-picker>`,
+    template: `<fd-date-picker [(ngModel)]="selectedDay" [placement]="'top-end'"></fd-date-picker>`,
```

Comment:
Again this probably does not belong in the i18n section but may warrant its own example